requirements.txt

```diff
@@ -1,7 +1,7 @@
-ovos_plugin_manager
-ovos_workshop
-ovos_utils
+ovos_plugin_manager~=0.0.25
+ovos_workshop~=0.0.15
+ovos_utils~=0.0,>=0.0.38
 ovos_classifiers
-pyjokes
-ovos-translate-server-plugin>=0.0.0a5
-neon-lang-plugin-libretranslate
+pyjokes~=0.6
+ovos-translate-server-plugin~=0.0.1
+neon-lang-plugin-libretranslate~=0.2
```

Comment:
this is no longer needed, it was a workaround for missing dependencies before the translate server was introduced